content/docs/dvclive/user-guide/quickstart.md

```diff
@@ -0,0 +1,98 @@
+# Quickstart
```

Comment:
it feels it's better to move the Quick start to the top out of the UserGuide (same as we do for CML and DVC).